sw

```diff
@@ -47,15 +47,15 @@ else
 fi
 
 # GNU date accepts the input date differently than BSD
-if [[ $USE_GNU_DATE == "0" ]]; then
+if [[ $USE_GNU_DATE == "1" ]]; then
```

Comment:
I would suggest a complete overhaul of the GNU 'date' detection.
What you hotfixed is the problem that your 'date', like mine, is a GNU 'date' which does not work with the '-v' option, and so you manipulate the if to use the GNU compatible options, but this does not solve the original problem.

A fix what I have in mind checks if 'date' is of a GNU nature or not before setting the final 'USE_GNU_DATE' variable. Also set the use of 'date' or 'gdate' in a new variable before the stopwatch actually starts.

If desired that I present my case in code, let me know and I will create a PR in the next days.